common/djangoapps/xblock_django/api.py

```diff
@@ -27,6 +28,15 @@ def disabled_xblocks():
     return XBlockConfiguration.objects.current_set().filter(enabled=False)
 
 
+# TODO: move to openedx.core.xblock.api??
```

Comment:
@kdmccormick we're adding new functions to the XBlock API module. Should we add it here, or in the new `openedx.core.xblock.api` v2 module?

Reply:
TODO: Consider using https://github.com/openedx/XBlock/blob/master/xblock/plugin.py instead.